src/App.js

```diff
@@ -1,5 +1,63 @@
+import { Console } from "@woowacourse/mission-utils";
+
 class App {
-  async run() {}
+  run() {
+    return Console.readLineAsync('덧셈할 문자열을 입력해 주세요.')
+      .then((input) => {
+        const result = this.calculateSum(input);
+        Console.print(`결과 : ${result}`);
+      });
+  }
```

Comment:
이런 식의 Console.readLineAsync 사용법은 처음 봤어요!😀 신기하긴 하지만 코드가 조금 깊어지는 것 같기도 해요😅
라이브러리를 보면 readLineAsync가 async 함수예요! 그래서 await 키워드를 앞에 붙여서 입력을 할 때까지 기다릴 수 있어요! 그러면 이런 식으로 바꿀 수 있을 것 같아요~
```suggestion
  async run() {
    const input = await Console.readLineAsync('덧셈할 문자열을 입력해 주세요.');
    const result = this.calculateSum(input);
    Console.print(`결과 : ${result}`);
  }
```

Reply:
감사합니다!! 또 하나 알았네요 😄